tests/tests_release_notes_workflow/test_release_notes.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import unittest
+from typing import Any
+from unittest.mock import patch
+
+from manifests.input_manifest import InputComponentFromSource, InputManifest
+from release_notes_workflow.release_notes import ReleaseNotes
+
+
+class TestReleaseNotes(unittest.TestCase):
+
+    @patch("release_notes_workflow.release_notes.ReleaseNotes.table")
+    @patch("release_notes_workflow.release_notes.ReleaseNotes.check")
+    @patch("manifests_workflow.input_manifests.InputManifest.from_file")
+    @patch('release_notes_workflow.component_release_notes.ComponentsReleaseNotes.from_component')
+    def test_check(self, *mocks: Any) -> None:
+        MANIFESTS = os.path.join(
+            os.path.dirname(__file__),
+            "..",
+            "..",
+            "manifests",
+        )
+        OPENSEARCH_MANIFEST = os.path.realpath(os.path.join(MANIFESTS, "templates", "opensearch", "default", "manifest.yml"))
+        manifest_file = InputManifest.from_file(open(OPENSEARCH_MANIFEST))
+        release_notes = ReleaseNotes(manifest_file, "2022-07-26")
+        for component in manifest_file.components.select():
+            if type(component) is InputComponentFromSource:
+                self.assertTrue(release_notes.check(component))
+                assert type(release_notes.check(component)) == list
```

Comment:
This doesn't actually test anything. This is supposed to test `ReleaseNotes#check`, you should be calling that method and asserting that it did whatever it was supposed to do.

Reply:
Modifies the tests.